Ningen Kakumei (1995) - 04.en.ass

```diff
@@ -489,22 +490,22 @@ Dialogue: 0,0:30:03.97,0:30:05.99,Default,,0,0,0,,Deciding that is your role. {k
 Dialogue: 0,0:30:06.47,0:30:07.99,Default,,0,0,0,,Don't forget it!
 Dialogue: 0,0:30:08.45,0:30:09.30,Default,,0,0,0,,Got it!
 Dialogue: 0,0:30:11.00,0:30:16.02,Sign,,0,0,0,,Aobaso, an apartment in Omori where Shinichi Yamamoto moved to {山本伸一が移り住んだ大森のアパート「青葉荘」}
-Dialogue: 0,0:30:25.16,0:30:27.95,Default,,0,0,0,,Toda-sensei gave me a scolding.
-Dialogue: 0,0:30:27.95,0:30:29.84,Default,,0,0,0,,It was my own fault.
-Dialogue: 0,0:30:29.84,0:30:32.13,Default,,0,0,0,,I must reflect on my actions.
-Dialogue: 0,0:30:32.13,0:30:33.81,Default,,0,0,0,,I have to believe in him
-Dialogue: 0,0:30:33.81,0:30:36.59,Default,,0,0,0,,and keep working hard, \Nno matter for how long.
-Dialogue: 0,0:30:36.59,0:30:37.69,Default,,0,0,0,,I must always reflect
-Dialogue: 0,0:30:37.69,0:30:39.56,Default,,0,0,0,,on my own wrongdoings
-Dialogue: 0,0:30:39.56,0:30:42.39,Default,,0,0,0,,to become a revolutionized person. {unsure, maybe "someone who can start a revolution"}
-Dialogue: 0,0:30:42.39,0:30:43.87,Default,,0,0,0,,Otherwise,
-Dialogue: 0,0:30:43.87,0:30:45.97,Default,,0,0,0,,you won't have anyone else to look at,
-Dialogue: 0,0:30:45.97,0:30:47.60,Default,,0,0,0,,except your own pitiful life.
-Dialogue: 0,0:30:47.60,0:30:55.79,Default,,0,0,0,, Nam-Myoho-Renge-Kyo...
+Dialogue: 0,0:30:25.16,0:30:27.95,Thought,,0,0,0,,Toda-sensei gave me a scolding.
+Dialogue: 0,0:30:27.95,0:30:29.84,Thought,,0,0,0,,It was my own fault.
+Dialogue: 0,0:30:29.84,0:30:32.13,Thought,,0,0,0,,I must reflect on my actions.
+Dialogue: 0,0:30:32.13,0:30:33.81,Thought,,0,0,0,,I have to believe in him
+Dialogue: 0,0:30:33.81,0:30:36.59,Thought,,0,0,0,,and keep working hard, \Nno matter for how long.
+Dialogue: 0,0:30:36.59,0:30:37.69,Thought,,0,0,0,,I must always reflect
+Dialogue: 0,0:30:37.69,0:30:39.56,Thought,,0,0,0,,on my own wrongdoings
+Dialogue: 0,0:30:39.56,0:30:42.39,Thought,,0,0,0,,to become a revolutionary. {unsure, maybe "someone who can start a revolution"}

```

Comment:
```suggestion
Dialogue: 0,0:30:39.56,0:30:42.39,Thought,,0,0,0,,to become a revolutionized person. {unsure, maybe "someone who can start a revolution", but from past lines more likely "someone who underwent revolution"}
```

I still think he means "someone who underwent revolution" (like in the show's title).